DQMOffline/Trigger/plugins/BPHMonitor.cc

```diff
@@ -13,16 +13,55 @@ BPHMonitor::BPHMonitor( const edm::ParameterSet& iConfig ) :
   folderName_             ( iConfig.getParameter<std::string>("FolderName") )
   , muoToken_             ( mayConsume<reco::MuonCollection>       (iConfig.getParameter<edm::InputTag>("muons")     ) )  
   , bsToken_              ( mayConsume<reco::BeamSpot>             (iConfig.getParameter<edm::InputTag>("beamSpot")))
+  , trToken_              ( mayConsume<reco::TrackCollection>             (iConfig.getParameter<edm::InputTag>("tracks")))
```

Comment:
hi @mtosi - my confusion is that some (the ones in BPHMonitor.cc for example) changes are already part of the master branch. I don't want to figure out what is reviewed and accepted already. 

I'm confused why these are showing up in your PR (even as a commit you made)?